Cargo.toml

```diff
@@ -8,45 +8,34 @@ license = "MIT"
 homepage = "https://github.com/snapview/tokio-tungstenite"
 documentation = "https://docs.rs/tokio-tungstenite/0.9.0"
 repository = "https://github.com/snapview/tokio-tungstenite"
-version = "0.9.0"
+version = "0.10.0-alpha.1"
```

Comment:
Given that this version of the PR is no longer depending on any alpha libraries, do we want to drop the "alpha" here and release this as `0.10.0`?